.travis.yml

```diff
@@ -1,10 +1,18 @@
 sudo: false
+
 language: go
 
 go:
-    - 1.2
+    - 1.3
     - tip
 
-install: go get -t ./...
+services:
+  - redis-server
 
-script: go test -v -short ./...
+install:
+  - make get-deps
+  # Move to the gopkg location that rather than the default clone location
+  - export GOPKG_DIR=$GOPATH/src/gopkg.in/throttled
+  - mkdir -p $GOPKG_DIR
+  - mv $TRAVIS_BUILD_DIR $GOPKG_DIR/throttled.v0
+  - cd $GOPKG_DIR/throttled.v0
```

Comment:
If it finds a Makefile, Travis runs `make` as the test script by default. That's not obvious so I've updated the configuration to be explicit.